<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomateTestWidget</class>
 <widget class="QWidget" name="AutomateTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>864</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1536</width>
    <height>864</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>750</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>测试计划名称</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="beginTest">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>开始测试</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="endTest">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>终止测试</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="resultQuery">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string> 测试结果查询</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="delTestProject">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>删除测试计划</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addTestProject">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>新增测试计划</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="parameterBinding">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>测试计划参数装订</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,0,2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="testSchematics_2">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="paramtab">
        <attribute name="title">
         <string>测试参数</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QStackedWidget" name="stackedWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>测试过程</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="linkConfiguration">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>链路配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="parameterSetting">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>参数设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startTesting">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>开始测试</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testing">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>正在测试</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_11">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="endOfTest">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>测试结束</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>状态信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="statusInfo"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
